<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>M_SetPNAdlg</class>
 <widget class="QDialog" name="M_SetPNAdlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>362</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>315</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>315</width>
    <height>1024</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>失网设置</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../bin/image/607logo-6464.png</normaloff>../bin/image/607logo-6464.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>311</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>56</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
    <property name="title">
     <string>信号源设置</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>63</y>
       <width>59</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>发射功率：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>22</y>
       <width>59</width>
       <height>37</height>
      </rect>
     </property>
     <property name="text">
      <string>中频带宽：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>235</x>
       <y>32</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>20</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>dBm</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>12</y>
       <width>131</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_width"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_power"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
    <property name="title">
     <string>采集时间设置</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>21</y>
       <width>31</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>s</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>ms</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="lineEdit_space_time">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>62</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_continu_time">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>29</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>58</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>采集间隔时间：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>25</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>采集持续时间：</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>2</y>
      <width>301</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
    <property name="title">
     <string>测试模式</string>
    </property>
    <widget class="QRadioButton" name="radioButton_single">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>24</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>单点</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_continuous">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>23</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>89</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>89</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>连续</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>305</width>
      <height>53</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>64</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_adv">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;楷体&quot;;</string>
        </property>
        <property name="text">
         <string>脉冲</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;楷体&quot;;</string>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
